ocaml/idl/datamodel.ml

```diff
@@ -6519,12 +6519,41 @@ end
 (** PCI devices *)
 
 module PCI = struct
+  let disable_dom0_access =
+    call ~name:"disable_dom0_access"
+      ~lifecycle:[(Published, rel_vgpu_tech_preview, "")]
+      ~doc:
+        "Hide a PCI device from the dom0 kernel. (Takes affect after next \
+         boot.)"
+      ~params:[(Ref _pci, "self", "The PCI to hide")]
+      ~allowed_roles:_R_POOL_OP ()
+
+  let enable_dom0_access =
+    call ~name:"enable_dom0_access"
+      ~lifecycle:[(Published, rel_vgpu_tech_preview, "")]
+      ~doc:
+        "Unhide a PCI device from the dom0 kernel. (Takes affect after next \
+         boot.)"
+      ~params:[(Ref _pci, "self", "The PCI to unhide")]
+      ~allowed_roles:_R_POOL_OP ()
+
+  let is_dom0_access_enabled =
+    call ~name:"is_dom0_access_enabled"
+      ~lifecycle:[(Published, rel_vgpu_tech_preview, "")]
+      ~doc:
+        "Check whether a PCI device is reachable from the dom0 kernel on boot."
```

Comment:
The methods use the word access, it makes more sense to use accessible rather than reachable:
```suggestion
       "Returns whether the PCI device is accessible from the dom0 kernel on boot."
```